content/terminal/custom-backend/widgets.json.md

```diff
@@ -22,6 +22,17 @@ import HeadTitle from '@site/src/components/General/HeadTitle.tsx';
 
 <HeadTitle title="Widgets.json | OpenBB Terminal Pro Docs" />
 
+# Widgets.json Overview
+
+The `widgets.json` file is essential for integrating custom widgets into the OpenBB Terminal Pro. It serves as a configuration file that connects your backend data to the widgets displayed in the application. Key components include:
+
+- **Basic Information**: Defines the widget's name, description, and API endpoint.
```

Comment:
- **Basic Information**: Defines the widget's name, description, and API endpoint that the data comes from.